editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -405,6 +451,19 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     return nodeMetadata;
   }
 
+  // Relies on being called at a point where the user has answered the Qs on the node.
+  function getAllowListAnswers(nodeId: string) {
+    const { data } = flow[nodeId];
+    const nodeFn = data?.fn || data?.val || null;
```

Comment:
nit: is `|| null` necessary here if you're checking for existence of `nodeFn && ` on the next line?

Reply:
Fixed here: 1a06af7fe335dc8d877d6c0680e3df5cfd4001d5